index.html

```diff
@@ -275,7 +279,9 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
           <code>"measure"</code> [[USER-TIMING-2]], <code>"navigation"</code>
           [[NAVIGATION-TIMING-2]], <code>"resource"</code>
           [[RESOURCE-TIMING-2]], <!-- TODO: add long task spec reference -->
-           and <code>"longtask"</code>.</p>
+           and <code>"longtask"</code>.  These specifications need to indicate
```

Comment:
This seems like the wrong place for this comment.

Reply:
I dont disagree.  This is talking about the attribute in the PerformanceEntry object, but this is the only place in the spec where we call out where these types come from and it is these specs that need to determine if they are available from the timeline. 

Do you have another suggestion?  I think maybe it should be in the "Extensions to the Prformance Interface" section?  It isn't a method specifically, but maybe just add a "Note" in the 3.1 section?